src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/ProjectSnapshotManagerBase.cs

```diff
@@ -11,6 +11,8 @@ internal abstract class ProjectSnapshotManagerBase : ProjectSnapshotManager
 
         public abstract void ProjectChanged(Project underlyingProject);
 
+        public abstract void ProjectChanged(ProjectSnapshotUpdateContext update);
```

Comment:
Where is this called? Or is that coming in the next PR?

Reply:
Next PR